### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by pklaschka**
Docs and stuff

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,59 @@
-# dir-tree
+# `@fliegwerk/dir-tree`
+
+![NPM](https://img.shields.io/npm/l/@fliegwerk/dir-tree)
+![npm (scoped)](https://img.shields.io/npm/v/@fliegwerk/dir-tree)
+![node-current (scoped)](https://img.shields.io/node/v/@fliegwerk/dir-tree)
+![Libraries.io dependency status for latest release](https://img.shields.io/librariesio/release/npm/@fliegwerk/dir-tree)
```
Comment (fussel178) [inline README.md:6]: Which dependencies? :smile:

File: src/dir-tree.ts [hunk 2]
```diff
@@ -3,6 +3,21 @@ import { parse, join, resolve, dirname } from 'path';
 import { constants } from 'fs';
 import { TreeElement } from './tree-element';
 
+/**
+ * Builds an object representation of the file tree, starting at the passed `path`.
+ *
+ * @param path The path to the root of the tree
```
Comment (fussel178) [inline src/dir-tree.ts:9]:
```suggestion
 * @param path the path to the root of the tree
```

**Comment by fussel178** [comment]
I've rebased your branch to fix merge conflicts.

**Review by fussel178 — CHANGES_REQUESTED** [review]
One little typo. Everything else LGTM :taco:

File: README.md (hunk 1 above)
Comment (pklaschka) [reply README.md:6]: Precisely 😉

**Review by fussel178 — APPROVED** [review]

**Pull request merged**